### PR TITLE
Refactor client generation

### PR DESCRIPTION
**Pull request by serboctor**
- Remove client.py
- Move services to __init__.py
- Move http functionality to http_client.
- Fix all affected tests.
  
Fixes https://github.com/Jumpscale/go-raml/issues/394

### CODE REVIEW

File: codegen/python/client.go [hunk 1]
```diff
@@ -15,24 +15,32 @@ import (
 )
 
 const (
-	clientNameRequests = "requests"
+	clientNameRequests       = "requests"
 	clientNameGeventRequests = "gevent-requests"
-	clientNameAiohttp  = "aiohttp"
+	clientNameAiohttp        = "aiohttp"
```
Comment (iwanbk) [inline codegen/python/client.go]: it seems you need to integrate `go fmt` into your editor

File: codegen/python/client_template.go [hunk 2]
```diff
@@ -34,21 +34,21 @@ func (c *Client) initTemplatesRequests() {
 		oauth2Name:  "oauth2_client_python",
 		initFile:    "./templates/python/client_initpy_python.tmpl",
 		initName:    "client_initpy_python",
-		mainFile:    "./templates/python/client_python.tmpl",
-		mainName:    "client_python",
+		httpClientFile: "./templates/python/httpclient_python.tmpl",
```
Comment (iwanbk) [inline codegen/python/client_template.go]: since it is already in `python` dir, we should remove the `python` word, replace it with `requests` is better

Comment (serboctor) [reply codegen/python/client_template.go]: I kept python in the name to be consistent with all other names. But yea, it is redundant anyway. I'll change it and may be later on we can rename all the templates.

File: codegen/python/client.go (hunk 1 above)
Comment (serboctor) [reply codegen/python/client.go]: yea, it doesnt do it automatically. I have to run it myself. Will try to configure it.

**Pull request merged**